### PR TITLE
user-account creation something else changes for frontstage

### PR DESCRIPTION
**Pull request by creativeappdevamit**
# What and why?

# How to test?

# Trello
https://trello.com/c/XQ2gnUbP/369-s37-user-account-creation-something-else

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -72,6 +72,7 @@ class Config(object):
     ACCOUNT_EMAIL_CHANGE_ENABLED = bool(strtobool(os.getenv('ACCOUNT_EMAIL_CHANGE_ENABLED', "False")))
     SHARE_SURVEY_ENABLED = bool(strtobool(os.getenv('SHARE_SURVEY_ENABLED', "True")))
     MAX_SHARED_SURVEY = int(os.getenv('MAX_SHARED_SURVEY', '50'))
+    TECHNICAL_MESSAGE_ENABLED = bool(strtobool(os.getenv('SHARE_SURVEY_ENABLED', "True")))
```
Comment (mfcrocker) [inline config.py:75]:
```suggestion
    TECHNICAL_MESSAGE_ENABLED = bool(strtobool(os.getenv('TECHNICAL_MESSAGE_ENABLED', "True")))
```

**Review by mfcrocker — CHANGES_REQUESTED** [review]
One slight bug

File: config.py (hunk 1 above)
Comment (creativeappdevamit) [reply config.py:75]: Thanks, it would have been a disaster 👍 , will change it

Comment (creativeappdevamit) [reply config.py]: fixed

**Review by mfcrocker — APPROVED** [review]

**Pull request merged**